.github/workflows/cmake-linux.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           - args: "-DOC_IPV4_ENABLED=ON"
           # insecure with IPv4
           - args: "-DOC_SECURITY_ENABLED=OFF -DOC_IPV4_ENABLED=ON"
+          # enable Push Notification
+          - args: "-DOC_COLLECTIONS_IF_CREATE_ENABLED=ON -DOC_PUSH_ENABLED=ON"
```

Comment:
Do you see the Checks tab, you already have a run for this setup - https://github.com/iotivity/iotivity-lite/runs/6665915581?check_suite_focus=true . :) It failed but that's not your fault, we have a problematic default setup in CMake (I'm trying to fix the default setup in https://github.com/iotivity/iotivity-lite/pull/272), so for now add `-DOC_TCP_ENABLED=OFF` and it should succeed.